lib/src/tasks/coverage/api.dart

```diff
@@ -70,12 +70,11 @@ class CoverageTask extends Task {
   /// If [html] is true, `genhtml` will be used to generate an HTML report of
   /// the collected coverage and the report will be opened.
   static Future<CoverageResult> run(List<String> tests,
-      {bool functional: defaultFunctional,
-      bool html: defaultHtml,
+      {bool html: defaultHtml,
       String output: defaultOutput,
       List<String> reportOn: defaultReportOn}) async {
-    CoverageTask coverage = new CoverageTask._(tests, reportOn,
-        html: html, output: output, functional: functional);
+    CoverageTask coverage =
+        new CoverageTask._(tests, reportOn, html: html, output: output);
```

Comment:
I'm no longer passing the `functional` flag through here (which was used to conditionally spin up the Selenium server). This is because I've passed that responsibility on to the project (they have to use the `SeleniumHelper` class to start and stop the server). The side effect of this is that projects with functional tests will have to run the selenium server on every test run, even if functional tests aren't being run. This will be addressed in the project refactor and I'm fine with it for now because it shouldn't be a ton of overhead.